csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3616,7 +3634,16 @@ Secondly, the program for all items of:
 
 * type `/definitions/version_t`: If any element doesn't match the semantic versioning, replace the all elements of type `/definitions/version_t` with the corresponding integer version. For that, CVRF CSAF converter sorts the items of `/document/tracking/revision_history` by `number` ascending according to the rules of CVRF. Then, it replaces the value of `number` with the index number in the array (starting with 1). The value of `/document/tracking/version` is replaced by value of `number` of the corresponding revision item. The match must be calculated by the original values used in the CVRF document.
```

Comment:
This comment is for line 3615 (I cannot comment on it in this PR):
```
Secondly, the program for all items of:
```
The meaning is unclear to me. Do you mean `Secondly, the program fulfils all items of:`?